frontend/src/components/quiz/questions/QuestionApply.tsx

```diff
@@ -80,6 +80,8 @@ export const QuestionApply: FC<QuestionApplyProps> = ({ values, setFieldValue })
         sx={{
           '& .MuiToggleButton-root': {
             'borderRadius': '4px',
+            'fontWeight': 'bold',
+            'fontSize': '16px',
```

Comment:
Do the same for the other `ToggleButtonGroup` and also remove size `size="large"` prop.

```suggestion
            'fontWeight': '700',
            'fontSize': '1rem',
```

Reply:
Fixed